File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        HashMap<Integer, Integer> counts = new HashMap<>();

        for (int i : nums) {
            counts.put(i, counts.getOrDefault(i, 0) + 1);
        }

        List<Integer> vals = new ArrayList<>(counts.keySet());

        Collections.sort(vals, (v1, v2) -> Integer.compare(counts.get(v2), counts.get(v1)));

        int[] ret = new int[k];
        for (int i = 0; i < k; i++) {
            ret[i] = vals.get(i);
        }

        return ret;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        s = {}
        i = 0
        for n in nums:
            if (target - n) in s:
                return [s.get(target-n), i]
            s[n] = i
            i = i + 1
        
        return [i, i]

        